packages/hash/frontend/src/shared/readonly-mode/context.tsx

```diff
@@ -0,0 +1,37 @@
+import { useRouter } from "next/router";
+import { createContext, ReactNode, useContext, useMemo } from "react";
+
+type ReadonlyModeInfo = {
```

Comment:
```suggestion
export type ReadonlyModeInfo = {
```

Reply:
Updated in [a74f0d5](https://github.com/hashintel/hash/pull/893/commits/a74f0d55f7c9b7806d3b5ab86f07c41f1644fe6a)